### PR TITLE
Test that the coordinator broadcasts spend txs

### PR DESCRIPTION
**Pull request by danielabrozzoni**
Needs https://github.com/revault/coordinatord/pull/46

### CODE REVIEW

File: tests/test_spend.py [hunk 1]
```diff
@@ -694,3 +694,32 @@ def test_revaulted_spend(revault_network, bitcoind, executor):
...
             == len(deposits)
         )
+
+
+# Test that the coordinator will broadcast our spends
+@pytest.mark.skipif(not POSTGRES_IS_SETUP, reason="Needs Postgres for servers db")
+def test_coordinator_broadcast(revault_network, bitcoind, executor):
+    """
+    Revault an ongoing Spend transaction carried out by the managers, under misc
+    circumstances.
```
Comment (darosior) [inline tests/test_spend.py:704]: Hmm no. :)

File: tests/test_spend.py [hunk 2]
```diff
@@ -694,3 +694,32 @@ def test_revaulted_spend(revault_network, bitcoind, executor):
...
+    """
+    CSV = 12
+    revault_network.deploy(2, 2, n_stkmanagers=1, csv=CSV)
+
+    # Simple case. Managers Spend a single vault.
```
Comment (darosior) [inline tests/test_spend.py:709]: Same here, comment seems unrelated

File: tests/test_spend.py [hunk 3]
```diff
@@ -694,3 +694,32 @@ def test_revaulted_spend(revault_network, bitcoind, executor):
...
+    vault = revault_network.fund(0.05)
+    revault_network.secure_vault(vault)
+    revault_network.activate_vault(vault)
+    revault_network.unvault_vaults_anyhow([vault])
+
+    for w in revault_network.participants():
+        w.stop()
```
Comment (darosior) [inline tests/test_spend.py:716]: Isn't there a helper for that?

**Comment by danielabrozzoni** [comment]
Fixed, and rebased on top of #365 to have the coord update commit :)

File: tests/test_spend.py [hunk 4]
```diff
@@ -386,8 +386,8 @@ def test_spends_conflicting(revault_network, bitcoind):
 @pytest.mark.skipif(not POSTGRES_IS_SETUP, reason="Needs Postgres for servers db")
 def test_spend_threshold(revault_network, bitcoind, executor):
     CSV = 20
-    managers_threshold = 3
-    revault_network.deploy(17, 8, csv=CSV, managers_threshold=managers_threshold)
+    managers_threshold = 2
```
Comment (darosior) [inline tests/test_spend.py:389]: *Comment about the commit message, not this line.*

> I'm not really sure why we would test with a deployment that
big (I'm fairly convinced it was just bad copypasta from another
test). This should reduce the amount of timeouts in CI.

My rationale was testing larger setups. It should be an explicitly dedicated test, instead so ACK for decreasing it.

**Review by darosior — COMMENTED** [review]
utACK 5075f2384fc57036b6f5db32d0659e921c76161c
